src/panels/lovelace/cards/hui-tile-card.ts

```diff
@@ -263,27 +262,29 @@ export class HuiTileCard extends LitElement implements LovelaceCard {
     const entityId = this._config.entity;
     const stateObj = entityId ? this.hass.states[entityId] : undefined;
 
-    const tileClasses = { vertical: Boolean(this._config.vertical) };
+    const contentClasses = { vertical: Boolean(this._config.vertical) };
 
     if (!stateObj) {
       return html`
-        <ha-card class="disabled">
-          <div class="tile ${classMap(tileClasses)}">
-            <div class="icon-container">
-              <ha-tile-icon class="icon" .iconPath=${mdiHelp}></ha-tile-icon>
-              <ha-tile-badge
-                class="badge"
-                .iconPath=${mdiExclamationThick}
-                style=${styleMap({
-                  "--tile-badge-background-color": `var(--red-color)`,
-                })}
-              ></ha-tile-badge>
+        <ha-card>
```

Comment:
No longer needs the disabled class?

Reply:
No. I was used to color the background in grey but now we use red badge for unknown entities.